### PR TITLE
Add project comment queries and mutations

### PR DESCRIPTION
**Pull request by kgolezardi**

### CODE REVIEW

File: review/core/interactors/project_review.py [hunk 1]
```diff
@@ -41,9 +51,9 @@ def get_or_create_project_review(project, reviewee):
     return project_review
 
 
-def get_project_review(user, id):
+def get_my_project_review(user, id):
```
Comment (smmoosavi) [inline review/core/interactors/project_review.py]: `get_project_review` is a better name.

File: review/core/interactors/project_review.py [hunk 2]
```diff
@@ -26,13 +26,23 @@ def save_project_review(project, reviewee, **kwargs):
     return project_review
 
 
-def get_all_project_reviews(user):
+def get_all_my_project_reviews(user):
```
Comment (smmoosavi) [inline review/core/interactors/project_review.py]: `get_all_project_reviews` is a better name

File: review/core/schema/save_project_comment_mutation.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        text = graphene.String()
+        rating = Evaluation()
+
+    project_comment = graphene.Field(ProjectCommentNode)
+
+    @classmethod
+    def mutate_and_get_payload(cls, root, info, **args):
+        project_review = get_node(args['project_review_id'], info, ProjectReview)
+        user = info.context.user
+
```
Comment (smmoosavi) [inline review/core/schema/save_project_comment_mutation.py]: check if the user is authenticated

File: review/core/interactors/project_comment.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+
+def can_review_project(user, project_review):
+    if not is_at_phase(Phase.PEER_REVIEW):
+        return False
+    return user in project_review.reviewers.all()
+
+
+def save_project_comment(project_review, reviewer, **kwargs):
+    project_comment = get_or_create_project_comment(project_review=project_review, reviewer=reviewer)
```
Comment (smmoosavi) [inline review/core/interactors/project_comment.py]: assert use is authenticated

Comment (kgolezardi) [reply review/core/interactors/project_comment.py]: I don't think it was necessary

**Pull request merged**